ops/hookcmds/_action.py

```diff
@@ -62,7 +62,7 @@ def format_result_dict(
         if parent_key:
             key = f'{parent_key}.{key}'
 
-        if isinstance(value, MutableMapping):
+        if isinstance(value, Mapping):
             value = cast('dict[str, Any]', value)
```

Comment:
Should this be:
```suggestion
            value = cast('Mapping[str, Any]', value)
```
?

Reply:
Ah that makes sense, I had assumed it was here because we wanted to narrow the content type of the mapping, but I guess it's narrowed to `Mapping[Any, Any]` here which satisfies the type checker.